src/kvstore/NebulaStore.h

```diff
@@ -43,13 +46,28 @@ class NebulaStore : public KVStore, public Handler {
     NebulaStore(KVOptions options,
                 std::shared_ptr<folly::IOThreadPoolExecutor> ioPool,
                 HostAddr serviceAddr,
-                std::shared_ptr<folly::Executor> workers)
+                std::shared_ptr<folly::Executor> deprecatedWorkers)
             : ioPool_(ioPool)
             , storeSvcAddr_(serviceAddr)
-            , workers_(workers)
             , raftAddr_(getRaftAddr(serviceAddr))
             , options_(std::move(options)) {
         CHECK_NOTNULL(options_.partMan_);
+
+        std::shared_ptr<folly::ThreadFactory> threadFactory =
```

Comment:
It is a good idea to separate the thread pool. For now, ioThreadPool is used for all thrift rpc server and RaftPart to send rpc request. IMO, the root cause is that follower is busy to `processAppendLogRequest` in ioThreadPool and leader is busy to `replicateLogs`. It will benefit a lot from start a new `raftClientPool_` and `raftWorkers`